modules/tinymce/src/themes/silver/test/ts/browser/editor/core/ContentLanguageHiddenTest.ts

```diff
@@ -0,0 +1,27 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.core.ContentLanguageHiddenTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: 'language | bold italic'
+  }, [ Theme ]);
+
+  it('Does not show the toolbar button if content_langs is not defined', () => {
+    const editor = hook.editor();
+    assert.throw(() => TinyUiActions.clickOnToolbar(editor, '[title="Language"]'));
+  });
+
+  it('Does not show menu item if content_langs is not defined', async () => {
+    const editor = hook.editor();
+    TinyUiActions.clickOnMenu(editor, 'button:contains("Format")');
+    await TinyUiActions.pWaitForUi(editor, '[role="menu"]');
+    assert.throw(() => TinyUiActions.clickOnUi(editor, '[role="menu"] [title="Language"]'));
```

Comment:
Is `throw` valid? I thought it was `throws`?

Reply:
I think `throw` is the deprecated/older version IIRC? `throws` is the recommended/documented version: https://www.chaijs.com/api/assert/#method_throws